Scripts/FindSimilarIncidentsByText/FindSimilarIncidentsByText.py

```diff
@@ -108,7 +122,12 @@ def parse_time(date_time_str):
 candidates = [x for x in candidates if len(x.get(INCIDENT_TEXT_FIELD, 0)) >= MIN_TEXT_LENGTH]
 
 # compare candidates to the orginial incident using TF-IDF
-similarity_vector = get_similar_texts(incident_text, map(lambda x: x[INCIDENT_TEXT_FIELD], candidates))
+candidates_text = map(lambda x: x[INCIDENT_TEXT_FIELD], candidates)
+if demisto.args()['preProcessText'] == 'true':
+    incident_text = pre_process_nlp(incident_text)
```

Comment:
Hoping I got it right:
When entering this condition, incident_text turns from a string to list (by pre_process_nlp)
It might cause bug in line 25? (list within a list)

Reply:
fixed